Formula/ocrmypdf.rb

```diff
@@ -0,0 +1,80 @@
+class Ocrmypdf < Formula
+  include Language::Python::Virtualenv
+
+  desc "Adds an OCR text layer to scanned PDF files"
+  homepage "https://github.com/jbarlow83/OCRmyPDF"
+  url "https://files.pythonhosted.org/packages/9b/c0/13a0ffb2184b85018b64466d1303ec1ceee05acff0b8ada16fdbf2ec101d/ocrmypdf-5.4.4.tar.gz"
+  sha256 "e523591c5d4e4a8cdeee2e9e24c3f631c8638a803dba74c5b8b274681784dbf4"
+
+  depends_on :python3
+  depends_on :x11 # Pillow needs XQuartz
```

Comment:
Are you sure? There's plenty of `Pillow`-using formulae that do not require `x11`. Possibly all of them.